addon/components/rdf-input-fields/input/edit.js

```diff
@@ -1,13 +1,52 @@
 import { action } from '@ember/object';
 import { guidFor } from '@ember/object/internals';
 import SimpleInputFieldComponent from '../simple-value-input-field';
+import { FORM, SHACL } from '@lblod/submission-form-helpers';
+import { next } from '@ember/runloop';
 
 export default class FormInputFieldsInputEditComponent extends SimpleInputFieldComponent {
   inputId = 'input-' + guidFor(this);
 
+  constructor() {
+    super(...arguments);
+
+    if (!this.value) {
+      this.setDefaultValue();
+      next(this, () => {
+        if (this.value) {
+          this.updateFieldValue();
+        }
+      });
+    }
+  }
+
   @action
   updateValue(e) {
     e.preventDefault();
+    this.updateFieldValue();
+  }
+
+  updateFieldValue() {
     super.updateValue(this.value && this.value.trim());
   }
+
+  /**
+   * Sets a default value on the field if the property `form:defaultValue` is defined in the
+   * field's configuration
+   */
+  setDefaultValue() {
```

Comment:
I would consider moving this logic, the retrieval of configuration, into the model for `Field`.